packages/www/docs/api-reference/vod/list.mdx

```diff
@@ -105,29 +105,13 @@ axios
     "createdAt": 1644546528636,
     "videoSpec": {
         "format": "mp4",
-        "tracks": [
-            {
-                "fps": 30,
-                "type": "video",
-                "codec": "h264",
-                "width": 1280,
-                "height": 720,
-                "bitrate": 1225660,
-                "duration": 309.966667,
-                "pixelFormat": "yuv420p"
-            },
-            {
-                "type": "audio",
-                "codec": "aac",
-                "bitrate": 127999,
-                "channels": 2,
-                "duration": 309.986395,
-                "sampleRate": 44100
-            }
-        ],
         "duration": 309.986393
     },
     "playbackId": "$PLAYBACK_ID",
     "downloadUrl": "https://livepeercdn.com/asset/$PLAYBACK_ID/video"
 }
 ```
+
```

Comment:
The  docs for this page is now here: https://github.com/livepeer/livepeer-studio-docs/blob/main/docs/references/vod/list.mdx
I can make the changes there and @gioelecerati as a reviewer